react-components/src/components/Architecture/SettingsButton.tsx

```diff
@@ -266,74 +281,78 @@ function createSlider(command: BaseSliderCommand, t: TranslateDelegate): ReactNo
   );
 }
 
-function createDropdownButton(command: BaseOptionCommand): ReactNode {
+function DropdownButtonComponent({ command }: { command: BaseOptionCommand }): ReactNode {
   // @update-ui-component-pattern
   const [isVisible, setVisible] = useState(true);
-  const [uniqueId, setUniqueId] = useState(0);
 
   useOnUpdate(command, () => {
     setVisible(command.isVisible);
-    setUniqueId(command.uniqueId);
   });
   // @end
 
   if (!isVisible) {
     return null;
   }
 
-  return (
-    <DropdownButton
-      key={uniqueId}
-      inputCommand={command}
-      placement={'bottom'}
-      usedInSettings={true}
-    />
-  );
+  return <DropdownButton inputCommand={command} placement={'bottom'} usedInSettings={true} />;
```

Comment:
```suggestion
  return <DropdownButton inputCommand={command} placement="bottom" usedInSettings />;
```
(made a similar comment below)